include/robot_state_publisher/joint_state_listener.h

```diff
@@ -62,6 +62,12 @@ class JointStateListener {
   /// Destructor
   ~JointStateListener();
 
+  // Default value for number of attempts to read robot_description from parameter server
+  static const int default_description_read_repetitions_;
```

Comment:
I think adding these two constants to the JointStateListener class breaks ABI, which would block merging into Kinetic. Since these are only used in main(), how about they be local variables there?